privileges-grantor/README.md

```diff
@@ -44,6 +42,54 @@ module "apiary-privileges-grantor" {
 
 ```
 
+The apiary-privileges-grantor lambda can be found in the public [maven repository](https://mvnrepository.com/artifact/com.expediagroup.apiary/apiary-privileges-grantor-lambda).
+The jars can be downloaded from the link provided above and uploaded to S3 via terraform as follows:
+
+```
+variable "pg_lambda_version" {
+  description = "Version of the Privilege Grantor Lambda."
+  type        = "string"
+  default     = "4.2.0"
+}
+
+data "aws_s3_bucket" "apiary-extensions" {
+  bucket = "pg-s3-bucket"
+}
+
+resource "null_resource" "apiary-privileges-grantor-jar" {
+  depends_on = ["data.aws_s3_bucket.apiary_extensions"]
+
+  provisioner "local-exec" {
+    command = <<CMD
+        curl -sLo apiary-privileges-grantor-core-${var.pg_lambda_version}.jar https://repo1.maven.org/maven2/com/expediagroup/apiary/apiary-privileges-grantor-core/${var.pg_lambda_version}/apiary-privileges-grantor-core-${var.pg_lambda_version}.jar && zip apiary-privileges-grantor-core-${var.pg_lambda_version}.zip apiary-privileges-grantor-core-${var.pg_lambda_version}.jar
```

Comment:
there is no need to zip it, lambda can take a jar or a zip

Reply:
updated